tests/bluetooth/bsim_bt/bsim_test_advx/src/main.c

```diff
@@ -829,7 +829,7 @@ static void test_advx_main(void)
 
 	k_sleep(K_MSEC(400));
 
-	printk("Disabling...");
+	printk("Disabling Extended Advertising...");
```

Comment:
These clauses of 
```
prinkt("<action>...");
err = action();
if (err) {
    goto exit;
}
printk("success.\n");
```

Could potentially be put into smaller functions so that the print messages are not duplicated like here, e.g.

```
bool adv_enable(void)
{
	printk("Enabling Periodic Advertising...");
	err = ll_adv_enable(handle, 1);
	if (err) {
		return false;
	}
	printk("success.\n");

	return true;
}

int main(void) 
{
	if (!adv_enable()) {
		goto exit;
	}
}
```

as I see you also have the same clauses multiple times.